### PR TITLE
Refactor/replace markdown

### LINKED ISSUE #1037: Replace decentraland-gatsby Markdown component

### PR DESCRIPTION
**Pull request by 1emu**
Closes #1037

### CODE REVIEW

File: src/components/Common/Text/Heading.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.Heading {
+  color: var(--text);
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  letter-spacing: normal;
+}
+
+.Heading--h1 {
```
Comment (andyesp) [inline src/components/Common/Text/Heading.css:9]: I wouldn't associate html tags to sizes. I'd change this to lg/md/sm :)

File: src/components/Common/Text/Heading.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+type Props = React.HTMLAttributes<HTMLHeadingElement> & {
+  className?: string
+  as?: HeadingTypes
+}
+
+export default function Heading(props: Props) {
+  const { as, className } = props
+  if (as === 'h1') return <h1 {...props} className={classNames('Heading', 'Heading--h1', className)} />
+  if (as === 'h3') return <h3 {...props} className={classNames('Heading', 'Heading--h3', className)} />
+  return <h2 {...props} className={classNames('Heading', 'Heading--h2', className)} />
```
Comment (andyesp) [inline src/components/Common/Text/Heading.tsx:18]: Same as above comment. I'd always render a h2 as default. I'd differentiate here the tag vs the size.

File: src/components/Common/Text/List.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import classNames from 'classnames'
+
+import './List.css'
+
+export type ListProps = (React.HTMLAttributes<HTMLUListElement> | React.HTMLAttributes<HTMLOListElement>) & {
```
Comment (andyesp) [inline src/components/Common/Text/List.tsx:7]: Is this export necessary? Are we using this props outside this component?

File: src/components/Common/Text/List.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+export default function List(props: ListProps) {
+  if (props.ordered) {
+    return <ol {...props} className={classNames('List', props.className)} />
+  } else {
+    return <ul {...props} className={classNames('List', props.className)} />
+  }
+}
+
+export type ListItem = React.LiHTMLAttributes<HTMLLIElement>
```
Comment (andyesp) [inline src/components/Common/Text/List.tsx:20]: Same as the other prop.

File: src/components/Common/Text/Text.tsx [hunk 5]
```diff
@@ -29,19 +30,19 @@ export default function Text({
...
-    </p>
+  const componentClassNames = classNames(
+    'Text',
+    `Text--size-${size}`,
+    `Text--weight-${weight}`,
+    `Text--color-${color}`,
+    `Text--style-${style}`,
+    className
+  )
+  return as === 'span' ? (
```
Comment (andyesp) [inline src/components/Common/Text/Text.tsx:43]: I think you can do something like...
```
const Component = as

return <Component className={componentClassNames}>{children}</Component>
```
Not sure exactly how `as` is typed.

File: src/components/GrantRequest/BudgetInput.css [hunk 6]
```diff
@@ -66,10 +66,8 @@
...
 
-.BudgetInput__Subtitle > .dg.Paragraph {
-  font-style: italic;
-  font-weight: 400;
-  font-size: 13px;
-  line-height: 17px;
-  color: var(--black-600);
+.BudgetInput__Subtitle {
+  font-style: italic !important;
+  font-weight: 400 !important;
```
Comment (ncomerci) [inline src/components/GrantRequest/BudgetInput.css:71]: shouldn't be this a var?

**Review by andyesp — APPROVED** [review]

**Pull request merged**